tools/lang-handler-aliases.js

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env node
```

Comment:
yep. I should mention that I rewrote this script again, in a later patch, as a grunt plugin..